src/Button/ZoomToExtentButton/ZoomToExtentButton.tsx

```diff
@@ -28,6 +31,20 @@ interface DefaultProps {
    * Default is false.
    */
   constrainViewResolution: boolean;
+  /**
+   * The extent `[minx, miny, maxx, maxy]` in the maps coordinate system or an
+   * instance of ol.geom.SimpleGeometry that the map should zoom to.
+   */
+  extent: number[] | OlSimpleGeometry;
+  /**
+   * The center `[x,y]` in the maps coordinate system or an
+   * instance of ol.coordinate that the map should zoom to if no extent is given.
+   */
+  center: OlCoordinate;
+  /**
+   *  The zoom level 'x' the map should zoom to  if no extent is given.
+   */
+  zoom: number;
```

Comment:
Please mark all three parameters extent, center and zoom as optional (e.g. `zoom?: number;`)